src/tests/JIT/Regression/JitBlue/Runtime_64125/Runtime_64125.tt

```diff
@@ -0,0 +1,90 @@
+﻿<#@ template debug="false" hostspecific="true" language="C#" #>
+<#@ assembly name="System.Core" #>
+<#@ import namespace="System.Linq" #>
+<#@ output extension=".cs" #>
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
Looks like our standard file header doesn't include this "See the ..." line anymore.

Reply:
Okay, copied this from a wrong place

Reply:
Updated.